dev-libs/libmelf/libmelf-0.4.0-r1.ebuild

```diff
@@ -11,30 +11,32 @@ SRC_URI="http://www.hick.org/code/skape/${PN}/${P}.tar.gz"
 LICENSE="Artistic"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
-IUSE=""
+IUSE="static-libs"
 
 DEPEND=""
 RDEPEND="${DEPEND}"
 
-src_prepare() {
-	# This patch was gained from the elfsign-0.2.2 release
-	epatch "${FILESDIR}"/${PN}-0.4.1-unfinal-release.patch
-	# Cleanup stuff
```

Comment:
put the comments back in (you can have comments in bash arrays)

Reply:
Thanks for the hint.  Never tried before.